### ISSUE TITLE
Loading "Some Blocks" extension broken on develop

### ISSUE THREAD

**Issue by paulkaplan** [LLK/scratch-gui#791]
### Expected Behavior

_Please describe what should happen_

Adding the "Some Blocks" extension should load the category/blocks into the block palette. 

### Actual Behavior

_Describe what actually happens_

Modal closes, but category doesn't appear. Accompanied by this error
```
[90mvm [39m[31merror[39m Error processing block: Cannot read property 'toString' of undefined, Block:
{}
[90mvm [39m[31merror[39m Failed to register primitives for extension on service extension.0.0: {}
```

### Steps to Reproduce

_Explain what someone needs to do in order to see what's described in *Actual behavior* above_

1. https://llk.github.io/scratch-gui/develop/
2. Click "Extensions"
3. Click "Some Blocks"
4. Error above

**Comment by paulkaplan** [comment]
The fix for this should include an update to the integration tests that opens the extensions modal and brings in an extension. That should be as simple as adding this test to `test/integration/test.js`
```
    test('Importing extensions', async () => {
        await driver.get(`file://${uri}`);
        await clickText('Blocks');
        await clickText('Extensions');
        await clickText('Some Blocks'); // Modal closes
        await clickText('Some Blocks', blocksTabScope); // Click the new category
        const logs = await getLogs(errorWhitelist);
        await expect(logs).toEqual([]);
    });
```

**Comment by paulkaplan** [comment]
Problem was resolved by #793, and a test was added.
